reference-architecture/osp-cli/docker-storage-setup

```diff
@@ -0,0 +1,2 @@
+DEVS=/dev/vdb
+VG=docker-vg
```

Comment:
I am not using Ansible (yet) to do anything but the openshift installation.

The point isn't to get the job done, it's to expose what's happening so that deployers have a chance to understand what can go wrong and what it might look like.

In the next revision the refarch authors for OCP on cloud are looking at replacing the python and bourne shell we're using now with more ansible, but only if it doesn't obscure the underlying change.